src/TodoList.jsx

```diff
@@ -0,0 +1,29 @@
+const todoList = [
+    {
+      id: 1,
+      title: 'Complete assignment',
+      date: '2020-02-15',
+    },
+    {
+      id: 2,
+      title: 'Submit assignment',
+      date: '2022-10-30',
+    },
+    {
+      id: 3,
+      title: 'Check assignment',
+      date: '2024-08-01',
+    },
+];
+
+export default function TodoList() {
+    return (
+        <ul>
+        {todoList.map(function (item) {
+          return (<> 
+      < li key={item.id}>{item.title}</li>
+        </>)
```

Comment:
I recommend looking into [Prettier ](https://prettier.io/) and getting that set up in VSCode to format this code. That way, it will keep you code clean and remove extraneous spaces like you have on line 24.